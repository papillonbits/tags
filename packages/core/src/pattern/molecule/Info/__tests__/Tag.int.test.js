import { regular } from './Tag.int.story'

describe('<Tag />', () => {
  describe('Render', () => {
    test('must match regular', () => {
      expect(global.renderToJSON(regular())).toMatchSnapshot()
    })
  })
})
