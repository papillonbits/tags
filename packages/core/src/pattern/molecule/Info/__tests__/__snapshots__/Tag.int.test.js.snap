// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tag /> Render must match regular 1`] = `
<div>
  <div
    class="tag"
  >
    <div
      class="tagText"
    >
      <span
        class="info infoSmall"
      >
        me@me.com
      </span>
    </div>
    <div
      class="tagImage"
    >
      <svg
        font-size="1px"
        height="12"
        viewBox="0 0 1024 1024"
        width="12"
      >
        <path
          d="M810.667 170.667q18.333 0 30.5 12.167t12.167 30.5q0 18-12.333 30.333l-268.667 268.333 268.667 268.333q12.333 12.333 12.333 30.333 0 18.333-12.167 30.5t-30.5 12.167q-18 0-30.333-12.333l-268.333-268.667-268.333 268.667q-12.333 12.333-30.333 12.333-18.333 0-30.5-12.167t-12.167-30.5q0-18 12.333-30.333l268.667-268.333-268.667-268.333q-12.333-12.333-12.333-30.333 0-18.333 12.167-30.5t30.5-12.167q18 0 30.333 12.333l268.333 268.667 268.333-268.667q12.333-12.333 30.333-12.333z"
        />
      </svg>
    </div>
  </div>
</div>
`;
